README.md

```diff
@@ -747,7 +776,8 @@ direct values to set the configuration options of the plugin without using a
 `ConfigMap`), or used to override these options as desired.
 
 These values are as follows:
-```
+
+```yaml
```

Comment:
This isn't strictly yaml. Could you provide an example of how this renders?

Reply:
See example [here](https://github.com/chipzoller/nvidia-k8s-device-plugin/tree/cz-readme-fixes?tab=readme-ov-file#setting-other-helm-chart-values).